Saving images crashes app

Saving a random picture crashes the app on ios.
How to reproduce:
Clicking on picture so its fullscreen, then clicking on the share symbol, then clicking on save image, then app crashes. The iphone is probably out of disk space, not sure if this is part of the problem.

while running current master on a 12.2 iOS simulator the issue persists (even with #806)

I'm getting this output

```

2020-06-22 17:36:15.205104+0200 deltachat-ios[35110:8532496] [MC] System group container for systemgroup.com.apple.configurationprofiles path is /Users/cyberta/Library/Developer/CoreSimulator/Devices/4C8258F4-EC7F-4DF5-894D-2E1D222BBB44/data/Containers/Shared/SystemGroup/systemgroup.com.apple.configurationprofiles
2020-06-22 17:36:15.347500+0200 deltachat-ios[35110:8541726] [MC] Filtering mail sheet accounts for bundle ID: chat.delta, source account management: 1
2020-06-22 17:36:15.347678+0200 deltachat-ios[35110:8532496] [MC] Reading from private effective user settings.
2020-06-22 17:36:15.355117+0200 deltachat-ios[35110:8541726] [MC] Filtering mail sheet accounts for bundle ID: chat.delta, source account management: 2
2020-06-22 17:36:46.759537+0200 deltachat-ios[35110:8532496] [default] [ERROR] Failed to determine whether URL /Users/cyberta/Library/Developer/CoreSimulator/Devices/4C8258F4-EC7F-4DF5-894D-2E1D222BBB44/data/Containers/Shared/AppGroup/51CFB2FE-C1FC-4411-A1C1-A150C7AEB084/messenger.db-blobs/1592838143.3224878.jpg (n) is managed by a file provider
2020-06-22 17:36:50.328350+0200 deltachat-ios[35110:8532496] [core] SLRemoteComposeViewController: (this may be harmless) viewServiceDidTerminateWithError: Error Domain=_UIViewServiceErrorDomain Code=1 "(null)" UserInfo={Terminated=disconnect method}
2020-06-22 17:36:53.545220+0200 deltachat-ios[35110:8532496] [Generic] Video /Users/cyberta/Library/Developer/CoreSimulator/Devices/4C8258F4-EC7F-4DF5-894D-2E1D222BBB44/data/Containers/Shared/AppGroup/51CFB2FE-C1FC-4411-A1C1-A150C7AEB084/messenger.db-blobs/1592838143.3224878.jpg cannot be saved to the photo library: Error Domain=AVFoundationErrorDomain Code=-11828 "Cannot Open" UserInfo={NSUnderlyingError=0x600003c70ba0 {Error Domain=NSOSStatusErrorDomain Code=-12847 "(null)"}, NSLocalizedFailureReason=This media format is not supported., NSURL=file:///Users/cyberta/Library/Developer/CoreSimulator/Devices/4C8258F4-EC7F-4DF5-894D-2E1D222BBB44/data/Containers/Shared/AppGroup/51CFB2FE-C1FC-4411-A1C1-A150C7AEB084/messenger.db-blobs/1592838143.3224878.jpg, NSLocalizedDescription=Cannot Open}
```

--- a/deltachat-ios/Info.plist
+++ b/deltachat-ios/Info.plist
@@ -51,7 +51,7 @@
 	<key>NSPhotoLibraryUsageDescription</key>
 	<string>Delta Chat will let you choose which photos from your library to send.</string>
 	<key>NSPhotoLibraryAddUsageDescription</key>
-	<string>Delta Chat uses this permission to save images to your photo library.</string>
+	<string>Delta Chat wants to save images to your photo library.</string>
 	<key>UIBackgroundModes</key>
 	<array>
 		<string>fetch</string>

--- a/deltachat-ios/Info.plist
+++ b/deltachat-ios/Info.plist
@@ -50,6 +50,8 @@
 	<string>Delta Chat uses your microphone to record and send voice messages and videos with sound.</string>
 	<key>NSPhotoLibraryUsageDescription</key>
 	<string>Delta Chat will let you choose which photos from your library to send.</string>
+	<key>NSPhotoLibraryAddUsageDescription</key>
+	<string>Delta Chat uses this permission to save images to your photo library.</string>
 	<key>UIBackgroundModes</key>
 	<array>
 		<string>fetch</string>

